### PR TITLE
Fix comment indentation issue

### PR DESCRIPTION
**Pull request by tugsbayasgalan**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #70227

fixes: https://github.com/pytorch/pytorch/issues/65452
Differential Revision: [D33251107](https://our.internmc.facebook.com/intern/diff/D33251107)

### CODE REVIEW

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33251107).

File: test/test_jit.py [hunk 1]
```diff
@@ -9731,6 +9731,18 @@ def foo():
         scripted = torch.jit.script(foo)
         self.assertTrue(scripted())
 
+    def test_comment_ignore_indent(self):
+        class Model(torch.nn.Module):
+            def __init__(self):
+    # ERROR  # noqa: E115
```
Comment (gmagogsfm) [inline test/test_jit.py:9737]:
```suggestion
    # useless comment that is not indented correctly  # noqa: E115
```

**Review by gmagogsfm — APPROVED** [review]

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33251107).

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33251107).

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33251107).

**Pull request closed without merging**